### PR TITLE
stm32f4 lorawan

### PR DESCRIPTION
**Pull request by Deryugin**
Implement simple diagnostic RN2483 interface

### CODE REVIEW

File: src/drivers/flash/stm32cube/stm32flash_conf.h [hunk 1]
```diff
@@ -8,14 +8,13 @@
 #ifndef SRC_DRIVERS_FLASH_STM32FLASH_CONF_H_
 #define SRC_DRIVERS_FLASH_STM32FLASH_CONF_H_
 
-
+#include <config/embox/driver/flash/stm32f4cube.h>
```
Comment (anton-bondarev) [inline src/drivers/flash/stm32cube/stm32flash_conf.h]: Can be used only 'f4' version?

File: src/drivers/serial/stm32_usart/stm_usart_cube_diag.c [hunk 2]
```diff
@@ -37,8 +37,6 @@
 
 #include <drivers/serial/uart_device.h>
 
-#include <drivers/serial/stm_usart.h>
```
Comment (anton-bondarev) [inline src/drivers/serial/stm32_usart/stm_usart_cube_diag.c]: Why is this removed?

Comment (Deryugin) [inline src/drivers/serial/stm32_usart/stm_usart_cube_diag.c]: It duplicates another header above.

**Comment by anton-bondarev** [comment]
LGTM
